tests/Shared/Infrastructure/Provider/TranslationsCatalogProviderTest.php

```diff
@@ -114,6 +114,32 @@ public static function provideTestGetCatalogVersionByPullRequest(): array
                 ),
                 9,
             ],
+            [
+                PullRequest::create(
+                    id: new PullRequestId(
+                        repositoryOwner: 'PrestaShop',
+                        repositoryName: 'PrestaShop',
+                        pullRequestNumber: '1'
+                    ),
+                    labels: ['label'],
+                    approvals: [],
+                    targetBranch: '9.0.X'
+                ),
+                8,
```

Comment:
```suggestion
                9,
```
Seems like the expected value should be 9 no?